Solutions/Zinc Open Source/Analytic Rules/ZincOctober2022_IP_Domain_Hash_IOC.yaml

```diff
@@ -183,5 +182,5 @@ entityMappings:
         columnName: AlgorithmEntity
       - identifier: Value
         columnName: FileHashEntity        
-version: 1.0.2
+version: 1.1.0
```

Comment:
Also could you please clarify why this version is incremented as 1.1.0 instead of 1.0.3.